modules/FindBasisUniversal.cmake

```diff
@@ -214,8 +254,6 @@ foreach(_component ${BasisUniversal_FIND_COMPONENTS})
                 # itself.
                 set_property(TARGET BasisUniversal::Encoder APPEND PROPERTY
                     INTERFACE_LINK_LIBRARIES BasisUniversal::Transcoder)
-                set_property(TARGET BasisUniversal::Encoder APPEND PROPERTY
-                    INTERFACE_COMPILE_DEFINITIONS "BASISU_NO_ITERATOR_DEBUG_LEVEL")
```

Comment:
This needs to be still present for older versions, right? One more reason to keep MSVC 2015 / 2017 on 1.15 and 1.16.

Reply:
It's still there, just moved to a `BasisUniversalEncoder_DEFINITIONS` variable that is appended to if OpenCL is found

Reply:
Ah you mean whether they still use that at all, I'll check

Reply:
Apparently, it's not needed for 1.16 anymore, but 1.15 still needs it. I'll add a comment so it'll removed if support for 1.15 is ever purged.